tests/unit/datarobot_drum/drum/test_args_parser.py

```diff
@@ -654,3 +669,41 @@ def test_read_triton_server_configs_success(
                 minimal_score_cmd_args,
                 is_env_variable,
             )
+
+
+class TestGetArgOptions:
+    @pytest.mark.parametrize(
+        "args",
+        ["score_args", "server_args", "fit_args", "perf_args", "validation_args", "push_args"],
+    )
+    def test_get_options(self, request, args):
+        args = request.getfixturevalue(args)
+        options = get_args_parser_options(args)
+        arg_option = CMRunnerArgsRegistry.get_arg_option(options, ArgumentsOptions.CODE_DIR)
+        assert options.code_dir
+        assert arg_option == options.code_dir
+
+    @pytest.mark.parametrize("args", ["fit_args", "push_args"])
+    def test_get_options_invalid(self, request, args):
+        args = request.getfixturevalue(args)
+        options = get_args_parser_options(args)
+        arg_option = CMRunnerArgsRegistry.get_arg_option(
+            options, ArgumentsOptions.RUNTIME_PARAMS_FILE
+        )
+        assert arg_option is None
+        assert arg_option not in options
```

Comment:
this assert is `assert None not in options`.  did you mean `assert "runtime_params_file" not in options`?

Reply:
ohh my bad, yes it `assert "runtime_params_file" not in options